tuf/data/types.go

```diff
@@ -26,6 +64,15 @@ func (k SigAlgorithm) String() string {
 
 const defaultHashAlgorithm = "sha256"
 
+// NotaryDefaultExpiries is the construct used to configure the default expiry times of
+// the various role files.
+var NotaryDefaultExpiries = map[RoleName]time.Duration{
+	RoleName("root"):      notary.NotaryRootExpiry,
```

Comment:
Don't need to do casts, Go works it out from the `map` definition, can just do `"root": notary.NotaryRootExpiry,`

https://play.golang.org/p/uzOhh7BDsN